crates/neon/src/types_impl/mod.rs

```diff
@@ -133,12 +168,38 @@ impl JsValue {
     }
 }
 
-/// The JavaScript `undefined` value.
+/// The type of JavaScript
+/// [`undefined`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#primitive_values)
+/// primitives.
+///
+/// # Example
+///
+/// ```
+/// # use neon::prelude::*;
+/// # fn test(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+/// // Extract the console.log function:
+/// let console: Handle<JsObject> = cx.global().get(&mut cx, "console")?;
+/// let log: Handle<JsFunction> = console.get(&mut cx, "log")?;
+///
+/// // The undefined value:
+/// let undefined = cx.undefined();
+///
+/// // Call console.log(undefined):
+/// log.call_with(&cx).arg(undefined).exec(&mut cx)?;
```

Comment:
This code could be simplified a bit with [`Object::call_method_with`](https://deploy-preview-942--neon-bindings-docs.netlify.app/neon/object/trait.object#method.call_method_with).

```suggestion
/// // Extract the console object:
/// let console: Handle<JsObject> = cx.global().get(&mut cx, "console")?;
///
/// // The undefined value:
/// let undefined = cx.undefined();
///
/// // Call console.log(undefined):
/// console.call_method_with(&mut cx, "log")?.arg(undefined).exec(&mut cx)?;
```